skills/ai-first-documentation/scripts/audit-docs-mechanical.ps1

```
@@ -0,0 +1,561 @@
+    param([string]$DecisionFilePath)
+    $map = @{}
+    if (-not (Test-Path -LiteralPath $DecisionFilePath)) {
+        return $map
+    }
+    try {
+        $raw = Get-Content -LiteralPath $DecisionFilePath -Raw -ErrorAction Stop
+        # CRLF-normalize
+        $raw = $raw -replace '\r\n', "`n" -replace '\r', "`n"
+        # Find all H3 headings of form "### {check-id}: {relative-path}"
+        $pattern = '(?m)^### ([^\n]+)'
+        $matches = [regex]::Matches($raw, $pattern)
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'matches' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)